apps/imgviewer.c

```diff
@@ -47,22 +49,36 @@ int height = 300;
 sprite_t img = {0};
 
 #define APPLICATION_TITLE "Image Viewer"
+#define IMGVIEWER_VERSION   "1.4.0" 
```

Comment:
Where does this version number come from? It looks like you took this from Bim, which does not follow the standards for applications in ToaruOS as it is a separate project.